man/systemd-debug-generator.xml

```diff
@@ -70,6 +70,37 @@
     <citerefentry><refentrytitle>systemd.generator</refentrytitle><manvolnum>7</manvolnum></citerefentry>.</para>
   </refsect1>
 
+  <refsect1>
+    <title>System Credentials</title>
+
+    <variablelist class='system-credentials'>
+      <varlistentry>
+        <term><varname>systemd.extra-unit.*</varname></term>
+
+        <listitem><para>Credentials prefixed with <literal>systemd.extra-unit.</literal> specify additional
+        units to add to the final system. Note that these additional units are added to both the initrd and
+        the final system. <varname>ConditionPathExists=!/etc/initrd-release</varname> can be used to make
+        sure the unit is conditioned out in the initrd.</para>
+
+        <xi:include href="version-info.xml" xpointer="v256"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>systemd.unit-dropin.*</varname></term>
+        <term><varname>rd.systemd.unit-dropin.*</varname></term>
```

Comment:
hmm, do you have an actual usecase for the `rd.` thing?

I mean, our usual units are carefully put together that they work reasonably both in the initrd and in regular invocation. We do not support initrd-conditionalization for static drop-ins or any others, so why support this here for credential-based drop-ins?

I think dropping this would simplify things and be easier to grok